protocol/common/message.go

```diff
@@ -244,7 +244,7 @@ func (m *Message) MarshalJSON() ([]byte, error) {
 		EnsName                  string                           `json:"ensName"`
 		DisplayName              string                           `json:"displayName"`
 		Image                    string                           `json:"image,omitempty"`
-		AlbumID                  string                           `json:"albumId,omitempty"`
+		ImageAlbumID             string                           `json:"imageAlbumId,omitempty"`
```

Comment:
This looks like a breaking change. Just raising awareness that this could break clients that don't expect `imageAlbumId` yet